Ginger/GingerCore/Drivers/SeleniumDriver.cs

```diff
@@ -5302,54 +5313,132 @@ private void DoGetRecordings()
                 IframeClicked = false;
                 while (IsRecording)
                 {
-                    InjectRecordingIfNotInjected();
-                    HandleIframeClicked();
-                    HandleRedirectClick();
-                    Thread.Sleep(500);
-                    // TODO: call JS to get the recording
-
-                    PayLoad PLgerRC = new PayLoad("GetRecording");
-                    PLgerRC.ClosePackage();
-                    PayLoad plrcRec = ExceuteJavaScriptPayLoad(PLgerRC);
-
-                    if (!PLgerRC.IsErrorPayLoad())
+                    try
                     {
-                        List<PayLoad> PLs = plrcRec.GetListPayLoad();
+                        InjectRecordingIfNotInjected();
+                        HandleIframeClicked();
+                        HandleRedirectClick();
+                        Thread.Sleep(1000);
+                        // TODO: call JS to get the recording
```

Comment:
why this sleep increase is needed?

Reply:
this sleep is required when user will switch the page the script is not injected in to new url and user will start recording, which is causing the loss of some action recording so just increased the time from 500 to 1000 it was already there